### ISSUE TITLE
NTR: 6-pyruvoyltetrahydropterin synthetase low (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8164]
New term request 6-pyruvoyltetrahydropterin synthetase low (f400):

**Comment by pnrobinson** [comment]
@MickeySegal

Should this term really be "circulating" -- see below

Reduced 6-pyruvoyltetrahydropterin synthase activity can be documented in [red blood cells](https://www.sciencedirect.com/topics/medicine-and-dentistry/erythrocyte).

**Comment by MickeySegal** [comment]
We hadn't specified where it is measured but RBC seems to be correct.

**Comment by pnrobinson** [comment]
@MickeySegal Adding new term: 
- ID: HP:6000575
- label: Reduced circulating 6-pyruvoyltetrahydropterin synthase activity
- parents: [Abnormal circulating enzyme concentration or activity (HP:0012379)](https://hpo.jax.org/app/browse/term/HP:0012379)
- pmid: PMID:3297709
- def: The activity of 6-pyruvoyltetrahydropterin synthase in the blood circulation is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8164
- synonyms:
